content/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,247 @@
+access external services.
+This task shows that these Kubernetes mechanisms for accessing external services continue to work with Istio.
+The only configuration step you must perform is to disable Istio's
+[mutual TLS](/docs/concepts/security/#mutual-tls-authentication).
+
+While the examples in this task use HTTP protocols,
+Kubernetes Services for egress traffic work with other protocols as well.
+
+{{< boilerplate before-you-begin-egress >}}
+
+*  Create a namespace for a source pod without Istio control:
+
+    {{< text bash >}}
+    $ kubectl create namespace without-istio
+    {{< /text >}}
+
+*  Start the [sleep]({{< github_tree >}}/samples/sleep) sample in the `without-istio` namespace.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n without-istio
+    {{< /text >}}
+
+*   To send requests, create the `SOURCE_POD_WITHOUT_ISTIO` environment variable to store the name of the source
+    pod:
+
+    {{< text bash >}}
+    $ export SOURCE_POD_WITHOUT_ISTIO=$(kubectl get pod -n without-istio -l app=sleep -o jsonpath={.items..metadata.name})
+    {{< /text >}}
+
+*   Verify that the Istio sidecar was not injected, that is the pod has one container:
+
+    {{< text bash >}}
+    $ kubectl get pod $SOURCE_POD_WITHOUT_ISTIO -n without-istio
+    NAME                     READY   STATUS    RESTARTS   AGE
+    sleep-66c8d79ff5-8tqrl   1/1     Running   0          32s
+    {{< /text >}}
+
+## Kubernetes ExternalName service to access an external service
+
+1.  Create a Kubernetes
+    [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname) service for `httpbin.org`:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    kind: Service
+    apiVersion: v1
+    metadata:
+      name: my-httpbin
+    spec:
+      type: ExternalName
+      externalName: httpbin.org
+      ports:
+      - name: http
+        protocol: TCP
+        port: 80
+    EOF
+    {{< /text >}}
+
+1.  Observe your service. Note that it does not have a cluster IP.
+
+    {{< text bash >}}
+    $ kubectl get svc my-httpbin
+    NAME         TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+    my-httpbin   ExternalName   <none>       httpbin.org   80/TCP    4s
+    {{< /text >}}
+
+1.  Access `httpbin.org` via the Kubernetes service's hostname from the source pod without Istio sidecar:
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD_WITHOUT_ISTIO -n without-istio -c sleep -- curl my-httpbin.default.svc.cluster.local/headers
+    {
+      "headers": {
+        "Accept": "*/*",
+        "Host": "my-httpbin.default.svc.cluster.local",
+        "User-Agent": "curl/7.55.0"
+      }
+    }
+    {{< /text >}}
+
+1.  Disable mutual TLS for the calls to the external service:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: DestinationRule
+    metadata:
+      name: my-httpbin
+    spec:
+      host: my-httpbin.default.svc.cluster.local
```

Comment:
Maybe it would be better to include the explicit `tlsSettings` with mode set to `DISABLE`. The docs aren't really clear what the default is, and I think there's been some discussion about possibly honoring the `tlsSettings` of any other existing `DestinationRule` on the lookup path if the selected `DestinationRule` has other things (like subsets) but no explicit `tlsSettings` defined.

Reply:
Agree, good point! Handled in https://github.com/istio/istio.io/pull/4710/commits/5df8d209be2b4a05790bcee5cd23db071ccf3f9c